.storybook/main.ts

```diff
@@ -1,7 +1,7 @@
 import type { StorybookConfig } from '@storybook/react-webpack5';
 
 const config: StorybookConfig = {
-  stories: ['../src/**/*.stories.mdx', '../storybook/**/*.stories.mdx'],
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.tsx', '../storybook/**/*.stories.mdx'],
```

Comment:
Stories should now be in `*.stories.[tsx|js]` format and mdx files should not include `.stories`